theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-contribution.ts

```diff
@@ -178,6 +186,10 @@ export class TraceViewerContribution extends WidgetOpenHandler<TraceViewerWidget
         registry.registerCommand(OpenTraceCommand, {
             execute: () => this.launchTraceServer()
         });
+        registry.registerCommand(OpenTraceWithPathCommand, {
```

Comment:
So, this command is executed by an external trigger and the path is known, right? There is no need to open the directory dialog. That's the main difference to OpenWithRootPathCommand  below, which opens the directory dialog with a given location.

Reply:
Yes, that is correct.